src/tracks/Pattern.cpp

```diff
@@ -65,6 +65,39 @@ Pattern::Pattern( InstrumentTrack * _instrument_track ) :
 	m_steps( MidiTime::stepsPerTact() )
 {
 	setName( _instrument_track->name() );
+
+	// Resize this track to be the same as existing tracks in the BB
+	// Only if there already exists at least another track in the BB
+	const TrackContainer::TrackList & tracks = m_instrumentTrack->trackContainer()->tracks();
```

Comment:
Try to limit line lengths to 80 characters ([coding conventions](https://github.com/LMMS/lmms/wiki/Coding-conventions)).